src/cosalib/builds.py

```diff
@@ -34,29 +34,26 @@ def __init__(self, workdir=None):
             raise Exception("Builds schema too new; please update cosa")
         # for now, since we essentially just support "1.0.0" and "0.0.1",
         # just dillute to a bool
-        self._legacy = (self._version._major < 1)
+        if self._version._major < 1:
+            err = f"Unsupported build metadata version {self._version._major}"
```

Comment:
Maybe let's just print `{self._version}` here so it prints the full version instead of just `0`?